__tests__/app.test.js

```diff
@@ -70,3 +70,22 @@ describe("api/reviews/:review_id", () => {
       });
   });
 });
+
+describe("/api/users", () => {
+  describe("GET", () => {
+    test("should return an array of objects, with each object having a property of username, name, and avatar", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then(({ body }) => {
+          expect(Array.isArray(body.users)).toBe(true);
```

Comment:
We don't technically need this line.  Line 83 infers that it is an array (forEach only works on arrays)